daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/trace_input.txt

```diff
@@ -0,0 +1 @@
+0

```

Comment:
I would suggest just making the test use the `Util.echoN()` function so that input data comes is read from stdin rather than a file. It simplifies the test a bit, avoids a file with just a zero in it, and avoids having to update the Rat.scala file.